lib/tasks/transform-files.js

```diff
@@ -44,6 +48,19 @@ export default async function modifyFiles(options) {
   }
 }
 
+async function handleModulePrefixMisMatch(options) {
+  if (options.isEmbroiderWebpack) return;
```

Comment:
Why are we skipping this when you have passed --isEmbroiderWebpack? 🤔 surely this will fail whenever anyone upgrades to vite regardless of where they start from? 

Also I think this should go in the main cli file as a task on its own rather than during transform-files task

Reply:
oh! I thought this was a target flag for some reason. I'll remove